docs/docs/functions/macros.md

```diff
@@ -143,8 +143,126 @@ Defines a new function.
 === "Code"
 
     ```clojure linenums="1"
+    (defn fn-name [fn-parms] ;; function-name followed by input parameters in '[]'
+        (Msg parm)           ;; function body that returns a value
```

Comment:
Usually `params` is the chosen abbreviation for parameters.

```suggestion
    (defn fn-name [fn-params] ;; function-name followed by input parameters in '[]'
        (Msg param)           ;; function body that returns a value
```

Reply:
Agreed, fixed.